LogMonitor/src/LogMonitor/EventMonitor.cpp

```diff
@@ -436,7 +436,7 @@ EventMonitor::PrintEvent(
             status = ERROR_INVALID_HANDLE;
         }
 
-        if (ERROR_INSUFFICIENT_BUFFER == (status = GetLastError()))
+        if (ERROR_INSUFFICIENT_BUFFER == (status = GetLastError()) || ERROR_EVT_UNRESOLVED_VALUE_INSERT == status)

```

Comment:
nitpick: I feel like this'll be easier to read if we pull out `status = GetLastError()` onto it's own line then the if statement, thoughts?